src/BaseExportAdapter.php

```diff
@@ -32,12 +35,14 @@ public function __construct(
         LoggerInterface $logger,
         DbConnection $connection,
         QueryFactory $simpleQueryFactory,
+        ResultWriter $resultWriter,
```

Comment:
`ResultWriter` sa povodne vytvaral priamo v `BaseExportAdapter`, ... zmenil som to a teraz sa injectuje cez constuctor, ... aby ho bolo mozne v pripade potreby rozsirit/vymenit.